uwblueprint/starter-code-v2
# Create release v1.11.1

## Notion ticket link

N/A



## Implementation description
* Followed [release instructions](https://github.com/uwblueprint/starter-code-v2#creating-a-release)
* Includes 1 commit from main: 6254d72c37587ef716f364b5ee7da311846c2861



## Steps to test
(Foregoing testing on every combination as changes purely consist of type annotations and `eslint-disable` comments. The chosen combination touches most of the modified code paths)

1. Generate the following combination using the CLI: TypeScript, GraphQL, MongoDB, opt-in auth, opt-in file storage
2. Run the application:
```
docker-compose up --build
```
3. Run linter on backend
```
docker exec -it scv2_ts_backend /bin/bash -c "yarn lint"
```

Aside from some Prettier errors caused by extra new lines, there should be no others reported. (`backend/typescript/models/index.ts` currently has an `import/prefer-default-export` error but I will disable this check in main). Will not address the Prettier errors as they can easily be fixed by running `yarn fix`.



## What should reviewers focus on?
* Are all changes from #149 included?


## Checklist
